website/karakara/templates/html/_base.mako

```diff
@@ -14,6 +14,7 @@
         
         <!-- Scripts -->
         ## TODO: Consider moving js to bottom of page - this may require some refactoring of inline js
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/paho-mqtt/1.0.1/mqttws31.min.js" type="text/javascript"></script>
```

Comment:
todo: add a local copy of this instead of hotlinking

Reply:
We are more or less thinking about deprecating most of the old website server-side generated templates.
Your new browser2 kind of obsoletes a lot of the need for those pages.
I'll have a think as to how to download this at install time. I can't remember right now how the mechanism works.
Currently the old web stuff only needs websockets for admin functions. So I may move this to be included only at the right time.